### PR TITLE
Feature about page

### PR DESCRIPTION
**Pull request by anrichard**
- Updated text and hyperlinks on about us pages.
- Added articles template and styling
- Added three articles pages:
   - RelevantLegislature.jsx
   - SourcesData.jsx
   - AboutData.jsx

### CODE REVIEW

**Comment by kevinli-work** [comment]
Can you clean this code up so it passes the linter?

**Comment by anrichard** [comment]
Yeah, i'm working on the errors in lint.

File: src/js/components/article/AboutData.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+export default class AboutData extends React.Component {
+    render() {
+        return (
+            <div className="usa-da-about-article">
+                <Header />
+                <div className="breadcrumbs">
+                  <ul>
+                    <li><a href="#" role="button" title="Home" aria-label="Home">Home</a></li>
+                    <svg class="usa-da-icon-angle-right" viewBox="0 0 512 512" aria-label="Arrow Pointing Right Icon" height="12"><title>Arrow Pointing Right Icon</title><g><path d="M143.5 434.8L304 257 143.8 77.3 143.4 6l225.2 250.5L144 506"></path></g></svg>
```
Comment (kevinli-work) [inline src/js/components/article/AboutData.jsx]: Why aren't we using the `Icon` components any more?

File: src/js/components/article/AboutData.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+                    <li>About the Data</li>
+                  </ul>
+                </div>
+                <div className="article-wrapper">
+                    <h1>About the Data</h1>
+                    <hr className="results-divider" />
+                    <p>Last year, the U.S. government spent $3.85 trillion. Do you ever wonder how all that money was used? Well, as you might imagine, it is a complicated story. It involves hundreds of government agencies and programs, thousands of companies and non-profits, and millions of individual citizens across the country and the globe. Despite this complex story, there is an abundance of data available that can help make sense of it all, and that data is offered here on USAspending.gov.</p>
+                    <p>USAspending.gov puts this federal spending data in your hands. In order to understand this data, it’s helpful to first understand how the data is structured and categorized. Below is a graphic showing the various levels of federal spending data that you will encounter on this site.</p>
+                    <img src="url(../../img/aboutdata-graphic@2x.png" alt="Data Structure" />
```
Comment (kevinli-work) [inline src/js/components/article/AboutData.jsx]: Can you fix this image `src`? Also be images should be served out of `img/xxx.png`.

File: src/js/components/article/AboutData.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                    <p>Awards are the final level. Once OMB appropriates the amount of funds that an agency may use, agencies begin to take actions to implement their programs, projects, or activities to make use of their appropriated funds through several types of awards.<br/>Agencies engage in partnerships with organizations through these award types:</p>
+                    <ul>
+                      <li>Contracts</li>
+                      <li>Grants</li>
+                      <li>Direct Payments</li>
+                      <li>Loans</li>
+                      <li>Insurance</li>
+                    </ul>
+                    <p>On USAspending.gov, you can view many of the details of these various awards, including the agency that issued the award, the organization that received the award, the amounts obligated over time, the general purpose of the award, among other attributes.</p>
+                    <p>View our <a href="#">Search and Download page</a> to explore this data.</p>
```
Comment (kevinli-work) [inline src/js/components/article/AboutData.jsx]: Where is this link supposed to go?

File: src/js/components/article/RelevantLegislature.jsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+import React from 'react';
+import Header from '../sharedComponents/header/Header';
+import Footer from '../sharedComponents/Footer';
+
+export default class RelevantLegislature extends React.Component {
+    render() {
+        return (
+            <div className="usa-da-about-article">
+                <Header />
+                <div className="breadcrumbs">
```
Comment (kevinli-work) [inline src/js/components/article/RelevantLegislature.jsx]: If we're going to have breadcrumbs on every page, please make this a component.

File: src/js/components/article/RelevantLegislature.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+                    <svg class="usa-da-icon-angle-right" viewBox="0 0 512 512" aria-label="Arrow Pointing Right Icon" height="12"><title>Arrow Pointing Right Icon</title><g><path d="M143.5 434.8L304 257 143.8 77.3 143.4 6l225.2 250.5L144 506"></path></g></svg>
+                    <li>Relevant Legislature</li>  
+                  </ul>
+                </div>
+                <div className="article-wrapper">
+                    <h1>Relevant Legislature</h1>
+                    <hr className="results-divider" />
+                    <p>The <a href="#">Federal Funding Accountability and Transparency Act of 2006 (FFATA)</a> was signed into law on September 26, 2006. The legislation required that federal contract, grant, loan, and other financial assistance awards of more than $25,000 be displayed on a publicly accessible and searchable website to give the American public access to information on how their tax dollars are being spent. In 2008, FFATA was amended by the <a href="#">Government Funding Transparency Act</a>, which required prime recipients to report details on their first-tier sub-recipients for awards made as of October 1, 2010.</p>
```
Comment (kevinli-work) [inline src/js/components/article/RelevantLegislature.jsx]: Can you verify where all the links on this page are supposed to go?

File: src/js/components/article/AboutData.jsx (hunk 1 above)
Comment (kevinli-work) [inline src/js/components/article/AboutData.jsx]: Also you can't use `class` in React, it's `className`.

File: src/js/components/article/SourcesData.jsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+                    <svg class="usa-da-icon-angle-right" viewBox="0 0 512 512" aria-label="Arrow Pointing Right Icon" height="12"><title>Arrow Pointing Right Icon</title><g><path d="M143.5 434.8L304 257 143.8 77.3 143.4 6l225.2 250.5L144 506"></path></g></svg>
+                    <li>Sources of Data</li>
+                  </ul>
+                </div>
+                <div className="article-wrapper">
+                    <h1>Sources of Data</h1>
+                    <hr className="results-divider" />
+                    <p>The U.S Treasury Department brings together data from hundreds of federal agencies and systems to make federal spending data readily available to the public. Federal agencies report this data, in a standard format, to Treasury and other government-wide systems.</p>
+                    <p>The diagram below provides an overview of the sources of the data that are used on USAspending.gov. This model shows how the data flows from the agency financial and awards systems to reach the public website.</p>
+                    <img src="url(../../img/information-flow.png" alt="Information Flow" />
```
Comment (kevinli-work) [inline src/js/components/article/SourcesData.jsx]: Please fix the image source.

File: src/js/components/about/AboutData.jsx [hunk 7]
```diff
@@ -15,15 +15,8 @@ export default class AboutData extends React.Component {
                 <div className="aboutdata-inner-wrap">
```
Comment (kevinli-work) [inline src/js/components/about/AboutData.jsx]: Can you provide verification that you want to launch this page with lorem ipsum on it?

File: src/js/components/article/RelevantLegislature.jsx (hunk 4 above)
Comment (anrichard) [inline src/js/components/article/RelevantLegislature.jsx]: Created a breadcrumb component (Breadcrumb.jsx)

File: src/js/components/article/RelevantLegislature.jsx (hunk 5 above)
Comment (anrichard) [inline src/js/components/article/RelevantLegislature.jsx]: Added links

File: src/js/components/about/AboutData.jsx (hunk 7 above)
Comment (anrichard) [inline src/js/components/about/AboutData.jsx]: Updated with content given by Geoff

**Pull request merged**